addons/svm/src/commands/send_token.rs

```diff
@@ -50,7 +50,7 @@ lazy_static! {
                     internal: false
                 },
                 amount: {
-                    documentation: "The amount, in lamports, of the token to send.",
+                    documentation: "The amount of tokens to send, in base unit.",
```

Comment:
I'm pretty sure that for SPL tokens, the smallest unit is still called lamports

Reply:
ha, that'd be confusing. do you have some docs to share?

Reply:
ahh, actually I was wrong. AI lead me astray 😓 

You can set the decimals when minting a token, so it's not lamports. What you have is good